clustered/integration-test/build.gradle

```diff
@@ -44,6 +44,13 @@ compileTestJava {
   options.forkOptions.executable = Jvm.current().javacExecutable
 }
 
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+checkstyle {
```

Comment:
This change is not needed, it is already configured like that in the root build file.